lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -375,7 +375,7 @@ namespace Js
 
     ScriptContext::~ScriptContext()
     {
-        Assert(isFinalized || !isInitialized);
+        Assert(isFinalized);
```

Comment:
Logically `isFinalized || !isInitialized` seems like a reasonable assert though. Maybe we should fix the state of those flags along error paths so that we could keep the assert and ensure that we've really handled all of the cases...